### PR TITLE
Add more cve related properties during Get operations

### PR DESCRIPTION
**Pull request by miteshvp**

### CODE REVIEW

**Comment by msrb** [comment]
My worry here is that the changes to the API could break graph-cve-sync job.

File: README.md [hunk 1]
```diff
@@ -303,10 +303,17 @@ Description: Get list of CVEs for E, EP, or EPV.
...
 {
-  "count": 1,
-  "cve_ids": [
-    "CVE-2018-0001"
-  ]
+	"cve_list": [{
+		"name": "numpy",
+		"description": "Some description here updated just now.",
+		"version": "11.0",
+		"status": null,
```
[the remark this answers is not in the document]
Comment (miteshvp) [reply README.md]: Status from NVD library. https://nvd.nist.gov/vuln

File: README.md [hunk 2]
```diff
@@ -319,10 +326,26 @@ Description: Get list of CVEs ingested on that particular date.
...
+		"fixed_in": null,
+		"cvss": "10.0",
+		"cve_id": "CVE-2018-0001",
+		"ecosystem": "pypi"
+	}, {
+		"name": "io.vertx:vertx-core",
+		"description": "Some description here updated just now.",
+		"version": "3.4.1",
+		"status": null,
+		"fixed_in": "3.4.2",
```
[the remark this answers is not in the document]
Comment (miteshvp) [reply README.md]: To make it less complex, we could resolve the fix available in the earliest version. Maintaining all versions is going to be difficult with new releases coming in.

**Comment by miteshvp** [comment]
> My worry here is that the changes to the API could break graph-cve-sync job.

If you think others can break the graph-cve-sync job, maybe we could just make the changes in `<bydate>` endpoint

**Comment by msrb** [comment]
@miteshvp or different proposal:

I can deploy gremlin and data-importer on dev cluster, sync all CVEs, and then deploy this PR-image there. People could then develop against the dev cluster endpoint and once we have clarity around requirements, we could make a decision on how to proceed (merge this to master, or do something different).

WDYT?

**Comment by miteshvp** [comment]
@msrb - Maybe we could come up with dummy data also. This was implemented based on the properties required from Clair side. But we do not have any concrete integration proposal yet. Till that time, this PR can stay open. Start Work again when we have those requirements documented.

File: src/cve.py [hunk 3]
```diff
@@ -296,8 +298,9 @@ def put(payload):
 # Get CVEs by date
 cve_nodes_by_date_script_template = """\
 g.V().has('vertex_label','CVE')\
-.has('modified_date', modified_date)\
-.values('cve_id').\
+.has('modified_date', modified_date).as('cve')\
+.in('has_cve').as('epv')\
+.select('cve','epv').by(valueMap())\
```
Comment (msrb) [inline src/cve.py]: Hmm, what does this query do exactly?

**Comment by msrb** [comment]
@miteshvp +1 on the changes, although I wasn't able to test this. I am getting:
```
{
  "count": 0, 
  "cve_list": []
}
```

Is that just a problem on side?

**Comment by msrb** [comment]
Feel free to merge if you're satisfied.

File: src/cve.py (hunk 3 above)
Comment (miteshvp) [reply src/cve.py]: It gets CVE by date, and also get what EPVs are affected. We need certain properties e.g. `name` and `version` to be provided to clair integration. Hence this query

**Comment by miteshvp** [comment]
> Is that just a problem on side?

It requires certain data to be present with a specific date field. Works for me because I have such data in my dev-cluster.

**Pull request merged**